tools/webpack-make

```diff
@@ -190,7 +190,7 @@ function generateDeps(makefile, stats) {
     lines.push("\trm -rf $(" + prefix + "_OUTPUTS) $(" + prefix + "_INSTALL) " + makefile);
     lines.push("all-local:: " + prefix);
     lines.push("\t@true");
-    lines.push("clean-local:: clean-" + prefix);
+    lines.push("distclean-local:: clean-" + prefix);
```

Comment:
Hmmm, so does ```make clean``` no longer work as expected for developers? The usual semantics for that is clean out all the build files, but leave the build system in place.